File: sample-java/src/main/java/net/dudmy/sample_java/MyApplication.java
package net.dudmy.sample_java;

import android.app.Application;

import net.dudmy.tutorialpop.TutorialPopHelper;

import java.util.ArrayList;
import java.util.List;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        // Create list of class name for show.
        List<String> tutorList = new ArrayList<>();
        tutorList.add(FirstActivity.class.getSimpleName());
        tutorList.add(SecondActivity.class.getSimpleName());

        // Initialize for using tutorPopupHelper.
        TutorialPopHelper.INSTANCE.prepareTutorial(getApplicationContext(), tutorList);
    }
}

File: sample-kotlin/src/main/java/net/dudmy/sample_kotlin/MyApplication.kt
package net.dudmy.sample_kotlin

import android.app.Application
import net.dudmy.tutorialpop.TutorialPopHelper

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        // Initialize for using tutorPopupHelper.
        TutorialPopHelper.prepareTutorial(applicationContext,
                // Create list of class name for show.
                listOf(FirstActivity::class.java.simpleName, SecondActivity::class.java.simpleName))
    }
}
